tsl/test/expected/dist_ddl.out

```diff
@@ -462,6 +462,8 @@ ALTER TABLE some_schema.disttable SET SCHEMA public;
 ALTER TABLE disttable SET SCHEMA some_unexist_schema;
 ERROR:  schema "some_unexist_schema" does not exist
 \set ON_ERROR_STOP 1
+-- OWNER TO
+ALTER TABLE disttable OWNER TO :ROLE_1;
```

Comment:
Don't we need to check that the command has be carried out properly across the entire cluster? Maybe setting to `ROLE_1` and doing a SELECT from disttable will be good enough to check that?